mmdet3d/datasets/nuscenes_dataset.py

```diff
@@ -137,6 +139,19 @@ def __init__(self,
                 use_external=False,
             )
 
+    def get_cat_ids(self):
```

Comment:
Make `for idx, info in enumerate(self.data_infos)` in CBGSDataset, `get_cat_ids` should accept an argument `idx` and use that `idx` to obtain the `class_sample_idx` of a single image to obtain. This is more general and could make `get_cat_ids` be used for `ClassBalancedDataset`.